sources/install.sh

```diff
@@ -2210,6 +2312,20 @@ function install_vulny-code-static-analysis() {
   add-test-command "vulny-code-static-analysis --help"
 }
 
+function install_brakeman() {
+  colorecho "Installing Brakeman"
+  gem install brakeman
+  add-history brakeman 
+  add-test-command "brakeman --help"
+}
+
+function install_semgrep() {
+  colorecho "Installing semgrep"
+  pip3 install semgrep
```

Comment:
Just for the sake of uniformity, could you switch to `python3 -m pip install` please?